src/effects/native/balanceeffect.cpp

```diff
@@ -0,0 +1,176 @@
+#include "balanceeffect.h"
+
+#include "util/defs.h"
+
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
This will remove the feature to turn the bypass knob to fully dry. This can be a valid use case, but I am unsure if it is important enough compared to a tight scale. 
If you like a 16 to 500 Hz scale more. I can change it.